instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -287,8 +315,11 @@ def _instrument(self, **kwargs):
         )
         duration_histogram = meter.create_histogram(
```

Comment:
the metric has been renamed and should also be emitted depending on the feature flag (the old one is `http.client.duration` with `ms` unit and the new one is  `http.client.request.duration` with `s` unit)

Reply:
Got it. Currently working on figuring out how this can work with the HTTP_DUP case.

Reply:
Done in latest push